src/Notifications/Notifications.csproj

```diff
@@ -9,8 +9,20 @@
     <RootNamespace>Dynamo.Notifications</RootNamespace>
     <AssemblyName>Notifications</AssemblyName>
   </PropertyGroup>
+  <ItemGroup>
+    <None Remove="View\NotificationUI.xaml" />
+    <None Remove="Web\ArtifaktElement-Regular.woff" />
+    <None Remove="Web\index.bundle.js" />
+    <None Remove="Web\index.html" />
```

Comment:
This is something we need to work on. I would love to avoid hard copy of the existing component.